LambdaEngine/Include/Game/ECS/Components/Rendering/ParticleEmitter.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+
+#include "ECS/Component.h"
+#include "ECS/Entity.h"
+
+namespace LambdaEngine
+{
+
+
+	struct ParticleEmitterComponent
+	{
+		enum Shape
```

Comment:
1. I think you can move the declaration outside of the component
2. See the code standard for enums. They should either be prefixed by E or F.
3. `CONE` and `SPHERE` are both set to 0. Is there a particular reason for this? If not, is there a reason you want to explicitly set their values at all?
4. The enum can be named something more specific to its use case, like `EParticleShape`

Reply:
Fixed, Thanks for tips!